sender-stackdriver/src/main/java/zipkin2/reporter/stackdriver/internal/AwaitableUnaryClientCallListener.java

```diff
@@ -28,16 +29,20 @@
 
   Object result; // guarded by this
 
+  long timeoutMs = 5000; // how long to wait for server response in milliseconds
```

Comment:
final unless testing.. if for testing, test it :)

Reply:
I was thinking of making it configurable, but I just couldn't find a way to easily pass all the way down the chain.